content/advanced/330_servicemesh_using_appmesh/integrate_observability/xray_trace.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "XRay Trace"
+date: 2020-01-27T08:30:11-07:00
+weight: 93
+draft: false
+---
+
+#### XRay Trace
+
+AWS X-Ray helps developers and DevOps engineers quickly understand how an application and its underlying services are performing. When it’s integrated with AWS App Mesh, the combination makes for a powerful analytical tool.
+
+Go to console, go to X-Ray, you should see the below in the Service Map
```

Comment:
same with this 'go to console' sentence (see comment on container_insight.md)

Reply:
Can you explain the service map a bit? This can be a little confusing for first-time users, seeing the various clients (LB healthchecks, I think?)

Reply:
updated with details